src/elements/flip-card/flip-card/flip-card.scss

```
@@ -0,0 +1,61 @@
+@use '../../core/styles' as sbb;
+
+// Box-sizing rules contained in typography are not traversing Shadow DOM boundaries. We need to include box-sizing mixin in every component.
+@include sbb.box-sizing;
+
+:host {
+  --sbb-flip-card-background-color: var(--sbb-color-cloud-alpha-80);
+  --sbb-flip-card-border-radius: var(--sbb-border-radius-4x);
+
+  position: relative;
+  display: block;
+}
+
+:host([data-flipped]) {
+  --sbb-flip-card-background-color: var(--sbb-color-midnight);
+
+  ::slotted(sbb-flip-card-summary) {
+    --sbb-flip-card-summary-opacity: 0;
+    --sbb-flip-card-summary-pointer-events: none;
+  }
+
+  ::slotted(sbb-flip-card-details) {
+    --sbb-flip-card-details-opacity: 1;
+    --sbb-flip-card-details-translate-y: 0;
+  }
+}
+
+.sbb-flip-card--wrapper {
+  position: relative;
+  display: flex;
+  flex-flow: wrap;
```

If the height is set on the host, the inner element does not grow with it. It should be enough to just add 100% height.
```suggestion
  height: 100%;
  flex-flow: wrap;
```